<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AriadneClass</class>
 <widget class="QMainWindow" name="AriadneClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1198</width>
    <height>859</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ariadne</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>241</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Sensor Connect Status</string>
    </property>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_4">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>센서 통신</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>64</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Platform</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>64</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>GPS</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>LiDAR</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>170</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>주행 시작</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>YOLO</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>SCNN</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_lidar_restart">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>RESTART</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_lidar_stop">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>STOP</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_yolo_stop">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>STOP</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_yolo_restart">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>110</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>RESTART</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_scnn_stop">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>140</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>STOP</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_scnn_restart">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>140</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>RESTART</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>241</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Mission</string>
    </property>
    <widget class="QPushButton" name="Btn_Mission0">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Mission0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_mission1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Mission1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_mission2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Mission2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_mission3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Mission3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_mission6">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Mission6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_mission7">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Mission7</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_mission8">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Mission8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_mission4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Mission4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_mission9">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>140</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Mission9</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_mission5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Mission5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_mission9_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>180</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Mission10</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_mission9_3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>220</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Mission11</string>
     </property>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>680</y>
      <width>691</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>80</y>
      <width>211</width>
      <height>191</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>PC Con</string>
     </attribute>
     <widget class="QPushButton" name="Btn_up">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>▲</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Btn_down">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>▼</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Btn_right">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>▶</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Btn_left">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>◀</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>94</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Btn_gearInput">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>120</y>
        <width>71</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>gear input</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>remoteCon</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>10</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>E-Stop</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>280</y>
      <width>211</width>
      <height>501</height>
     </rect>
    </property>
    <property name="title">
     <string>SensorStatus</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>64</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>STEER</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>64</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>GEAR</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Encoder</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>64</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>AorM</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>64</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>E-Stop</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>64</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>BREAK</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>64</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>SPEED</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_4">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>120</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_5">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>150</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_6">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>180</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_7">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>210</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Latitude</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>300</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Longitude</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_8">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>260</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_9">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>290</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>330</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Heading</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_10">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>320</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>350</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>370</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>LeftLeftLine</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>400</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>LeftLine</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>430</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>RightLine</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>460</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>RightRightLine</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_11">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>370</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_12">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>400</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_13">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>430</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_14">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>460</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QCustomPlot" name="rt_plot" native="true">
    <property name="geometry">
     <rect>
      <x>269</x>
      <y>19</y>
      <width>691</width>
      <height>651</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>241</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>YOLO</string>
    </property>
   </widget>
   <widget class="QLabel" name="AriadneLogo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1198</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Ariadne.qrc"/>
 </resources>
 <connections/>
</ui>
